winui/Rating/ToolTip.md

```diff
@@ -1,12 +1,92 @@
 ---
 layout: post
 title: ToolTip with WinUI Rating control | Syncfusion
-description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
+description: Learn here all about Enable tooltip features and tooltip format support in the rating items in WinUI rating control.
 platform: WinUI
```

Comment:
@Priyanka-Vijayasankar Learn here all about how to enable the tooltip and format the tooltip content in WinUI Syncfusion Rating (SfRating) control.

Reply:
Changed the content as mentioned above.